ASFAchievementManager/ASF-Achievement-Manager.csproj

```diff
@@ -1,25 +1,21 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
     <Authors>Ryzhehvost</Authors>
-    <AssemblyVersion>0.1.2.0</AssemblyVersion>
-    <TargetFrameworks>netcoreapp3.1;net48</TargetFrameworks>
+    <AssemblyVersion>0.1.3.0</AssemblyVersion>
+    <TargetFrameworks>net5.0;net48</TargetFrameworks>
     <Nullable>enable</Nullable>
     <LangVersion>latest</LangVersion>
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="JetBrains.Annotations" Version="*" IncludeAssets="compile" />
```

Comment:
Why add a dependency on `Jetbrains.Annotations`? I didn't see a single using-statement.

Reply:
Another weird one. Back in Novemember when I forked this to work on a resolution to the ASF 5 upgrade I had issues with references not working properly, so I had to change the csproj to the recommended format from the ASF wiki, this in turn required directly referencing the `JetBrains.Annotations` package - it's used elsewhere in exisitng code.

However, I have now reverted the csproj back to it's prior state and instead just updated the `AssemblyVersion` and `TargetFrameworks` to support net5.0 and it appears to work - so probably just an issue with my machine when I forked this to work on originally.